Model/EasyshipCarrier.php

```diff
@@ -207,15 +208,16 @@ protected function _createEasyShipRequest(RateRequest $request)
                 }
 
                 $itemQty = (int)$item->getQty();
+                $unitPriceInclTaxAndDiscount = ($item->getRowTotal() - $item->getDiscountAmount() + $item->getTaxAmount()) / $itemQty;
```

Comment:
`getRowTotal`, `getDiscountAmount` and `getDiscountAmount` are valid methods?
Only used here in this repo.

Reply:
Yes. We tested them in our own test stores.